packages/user/src/components/UsersTable/AllUsersTable.tsx

```diff
@@ -0,0 +1,253 @@
+import { useTranslation } from "@dzangolab/react-i18n";
+import { DataTable } from "@dzangolab/react-ui";
+import { FilterMatchMode } from "primereact/api";
+import { ButtonProps } from "primereact/button";
+import { ColumnProps } from "primereact/column";
+import { Tag } from "primereact/tag";
+import { IconType } from "primereact/utils";
+
+import { InvitationModal } from "../Invitation";
+
+import type {
+  AdditionalInvitationFields,
+  AddInvitationResponse,
+  InvitationAppOption,
+  InvitationRoleOption,
+} from "@/types";
+import { InvitationActions } from "../Invitation/InvitationActions";
+
+type VisibleColumn =
+  | "name"
+  | "email"
+  | "roles"
+  | "signedUpAt"
+  | "app"
+  | "invitedBy"
+  | "status"
+  | "actions";
+
+export type AllUsersTableProperties = {
+  additionalInvitationFields?: AdditionalInvitationFields;
+  apps?: Array<InvitationAppOption>;
+  className?: string;
+  columns?: Array<ColumnProps>;
+  extraColumns?: Array<ColumnProps>;
+  fetchUsers?: (arguments_?: any) => void;
+  id?: string;
+  inviteButtonIcon?: IconType<ButtonProps>;
+  loading?: boolean;
+  onInvitationAdded?: (response: AddInvitationResponse) => void;
+  onInvitationResent?: (data: any) => void;
+  onInvitationRevoked?: (data: any) => void;
+  prepareInvitationData?: (data: any) => any;
+  roles?: Array<InvitationRoleOption>;
+  showInviteAction?: boolean;
+  showAppColumn?: boolean;
+  totalRecords?: number;
+  users: Array<object>;
+  visibleColumns?: VisibleColumn[];
+};
+
+export const AllUsersTable = ({
+  additionalInvitationFields,
+  apps,
+  className = "table-users",
+  columns,
+  extraColumns = [],
```

Comment:
lets update this to `additionalColumns` to make it consistent with additionalFields